CaPPMS/Pages/Faq.razor

```diff
@@ -1,7 +1,203 @@
+
+                <span class="fas" style="color:black;font-weight:bold"> Add Question: </span>
+                <InputText @bind-Value=" NewFaq.Question" /><br />
+
+                <span class="fas" style="color:black;font-weight:bold"> Add Author: </span>
+                <InputText @bind-Value=" NewFaq.Author" /><br />
+
+                @*<span class="fas" style="color:black;font-weight:bold"> Add Reply: </span>
+                <InputTextArea @bind-Value=" NewFaq.Reply" /><br />*@
+
+                <button type="submit" class="btn-primary">Add Faq</button>
+            </div>
+        </EditForm>
+    }
+</div>
+
+
+<div class="m-2 p-2 rounded">
+    @*@foreach (var faq in faqManager.GetFaqs)
+    {
+
+        <table border="1">
+            
+            <tr>
+            <th> Newly Added FAQ </th>
+            </tr>
+                
+            <tr class="faqinputs">
+                <td class="fas" style="color:black;font-weight:bold">
+                    Topic:
+
+                </td>
+                @faq.Topic
+            </tr>
+
+            <tr class="faqinputs">
+                <td class="fas" style="color:black;font-weight:bold">
+                    Question:
+
+                </td>
+                @faq.Question
+            </tr>
+
+            <tr class="faqinputs">
+                <td>
+                    Author:
+
+                </td>
+                @faq.Author
+            </tr>
+
+            <tr class="faqinputs">
+                <td class="fas" style="color:black;font-weight:bold">
+                    Reply:
+
+                </td>
+                @faq.Reply
+            </tr>
+        </table>*@
+    @* }*@
+</div>
+<div>
+    @foreach (var faq in faqManager.GetFaqs)
+    {
+        <EditForm Model="NewFaq" OnValidSubmit="@HandleValidSubmit">
+            <DataAnnotationsValidator />
+            <ValidationSummary />
+            <div class="card" id="@faq.Guid">
+
+                <span class="fas" style="color:black;font-weight:bold"> Topic:  </span>
+                @*<InputText @bind-Value=" NewFaq.Topic" /><br />*@
+                    <span class="fas" style="color:black"/>
+                    <span> @faq.Topic </span><br />
+
+                    <span class="fas" style="color:black;font-weight:bold"> Question:</span>
+                    <span> @faq.Question </span><br />
+                    @*<InputText @bind-Value=" NewFaq.Question" /><br />*@
+
+                    <span class="fas" style="color:black;font-weight:bold"> Author:  </span>
+                    @*<InputText @bind-Value=" NewFaq.Author" /><br />*@
+                    <span> @faq.Author </span><br />
+
+                    @*<span class="fas" style="color:black;font-weight:bold"> Reply: @faq.Reply </span>
+            <InputTextArea @bind-Value=" NewFaq.Reply" /><br />
+
+            <button type="submit" class=" btn ">Submit Reply</button>*@
+            </div>
+        </EditForm>
+    }
+</div>
+
+<style>
```

Comment:
Style doesn't normally go in the page: https://docs.microsoft.com/en-us/aspnet/core/blazor/components/css-isolation?view=aspnetcore-5.0